platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/ContextMenu.kt

```diff
@@ -93,3 +116,29 @@ public object ContextMenuDivider : ContextMenuItem("---", {})
 
 public class ContextSubmenu(label: String, public val submenu: () -> List<ContextMenuItem>) :
     ContextMenuItem(label, {})
+
+public class ContextMenuOption(
+    public val icon: IconKey? = null,
+    public val actionType: ContextMenuItemActionType,
+    label: String,
+    action: () -> Unit,
+) : ContextMenuItem(label, action)
+
+public enum class ContextMenuItemActionType {
```

Comment:
I am a bit worried that this is putting us in a corner. What if in the future we want to add something "Copy as plain text" or "Paste as plain text"? Would this be possible to do without breaking changes?